src/App.tsx

```diff
@@ -1,39 +1,63 @@
 import React from 'react';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list